### PR TITLE
update environments

### PR DESCRIPTION
**Pull request by katamaks**

### CODE REVIEW

**Comment by katamaks** [comment]
@jtwaleson "Activity types" must be filled in...need a little help here

**Comment by katamaks** [comment]
@jtwaleson activity types ready for review

File: content/developerportal/deploy/environments.md [hunk 1]
```diff
@@ -94,41 +102,46 @@ After clicking [Details](/developerportal/deploy/environments-details) of the se
...
+Backup Restore Started | a backup restore has been manually started
+Backup Restored | the backup restore has been restored
+Clean | the environment has been cleaned
+Custom Domain | changes were made to the custom domain
+Database Backup Downloaded | the database backup has been downloaded
+Migration | the app has been migrated from one node to another node within the same Mendix Cloud version
+Error | error has occured while uploading a '.mda' file
+Files Backup Downloaded | backup files have been downloaded ?
+Maintenance | the admin password has been changed
+MDA Uploaded | a '.mda' file has been uploaded
```
Comment (jtwaleson) [inline content/developerportal/deploy/environments.md]: a -> an

File: content/developerportal/deploy/environments.md [hunk 2]
```diff
@@ -94,41 +102,46 @@ After clicking [Details](/developerportal/deploy/environments-details) of the se
...
+------------ | -------------
+Backup Created | a backup has been manually created
+Backup Expired | a backup has been expired after its expiration period
+Backup Restore Started | a backup restore has been manually started
+Backup Restored | the backup restore has been restored
+Clean | the environment has been cleaned
+Custom Domain | changes were made to the custom domain
+Database Backup Downloaded | the database backup has been downloaded
+Migration | the app has been migrated from one node to another node within the same Mendix Cloud version
+Error | error has occured while uploading a '.mda' file
```
Comment (jtwaleson) [inline content/developerportal/deploy/environments.md]: a -> an

File: content/developerportal/deploy/environments.md [hunk 3]
```diff
@@ -94,41 +102,46 @@ After clicking [Details](/developerportal/deploy/environments-details) of the se
...
+Files Backup Downloaded | backup files have been downloaded ?
```
Comment (jtwaleson) [inline content/developerportal/deploy/environments.md]: remove '?'

File: content/developerportal/deploy/environments.md [hunk 4]
```diff
@@ -94,41 +102,46 @@ After clicking [Details](/developerportal/deploy/environments-details) of the se
...
+Activity type | Explanation
+------------ | -------------
+Backup Created | a backup has been manually created
+Backup Expired | a backup has been expired after its expiration period
+Backup Restore Started | a backup restore has been manually started
+Backup Restored | the backup restore has been restored
+Clean | the environment has been cleaned
+Custom Domain | changes were made to the custom domain
+Database Backup Downloaded | the database backup has been downloaded
+Migration | the app has been migrated from one node to another node within the same Mendix Cloud version
```
Comment (jtwaleson) [inline content/developerportal/deploy/environments.md]: Remove 'within the same Mendix Cloud version'. We might want to use this for v3 -> v4 migrations in the future.

**Comment by katamaks** [comment]
@jtwaleson please approve

**Pull request merged**